tests/vmi_networking_test.go

```diff
@@ -722,22 +712,24 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				tcpPort := 8080
 				tests.StartTCPServer(serverVMI, tcpPort)
 
+				var ipAddr string
 				if networkCIDR == "" {
-					networkCIDR = api.DefaultVMCIDR
+					ipAddr = gatewayIPFromCIDR(api.DefaultVMCIDR)
+				} else {
+					ipAddr = gatewayIPFromCIDR(networkCIDR)
```

Comment:
The original code seems fine to me... and shorter.

Reply:
I thought about that .... And it would work, since it is not a pointer, but... Idk, changing the value of the input parameter mid-test seems awfully ... awful ? A bad practice at least.

Never the less, you're holding this, and I am not a zealot. Whatever you prefer here; this is more code, but avoids changing the input parameter; the other is less code, but changes the input parameter.

Reply:
You are welcome to convince me there is such a rule somewhere. For me, the input parameter is normalized at this point.
The fact that the name fits perfectly is a sign to me that it is ok.

Reply:
I choose to not engage; have it your way.